Artesian/Artesian.SDK/Factory/ActualTimeSerie.cs

```diff
@@ -105,6 +140,42 @@ private AddTimeSerieOperationResult _add(LocalDateTime localTime, double? value)
             return AddTimeSerieOperationResult.ValueAdded;
         }
 
+        private AddTimeSerieOperationResult _addRange(Dictionary<LocalDateTime, double?> values)
+        {
+            var duplicateKeys = _values.Keys.Intersect(values.Keys).ToList();
+
+            if (duplicateKeys.Any())
```

Comment:
I'm not convinced about this handling in a bulk case.

If even 1 record is already present, all are skipped.

While on the existing interface it does make sense (first one wins, no overwrite) and could evolve in overwrite with an additional control parameter, this implementation is more dangerous are data that the Client "adds" get's ignored while would not happen if foreach and adding one by one wouldn't.

The behaviour of the bulk and foreach are much different here and that is not even documented.

Please refine this. I don't think this is sane and the return value is misleading for a bulk case.